### PR TITLE
replaced npm install with bun i

### LINKED ISSUE #114: [Documentation Update]: Replace NPM install with bun i for setting up test cases.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Description

IN- Documentation for test inside store folder it is told to use NPM for installing dependencies where as the project uses bun to install dependencies 

### Suggested Change

IN readme.md inside frontend/lib/store/tests NPM is being used to install dependencies. Replace it with bun.

### Rationale

To use bun instead of NPM for installing dependencies.

### Urgency

Medium

### Record

- [X] I have read the Contributing Guidelines
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Dedsec933**
# Pull Request for Memfree

## Description

Replace NPM install with bun i for setting up test cases.

## Related Issue

If this pull request addresses an issue, please link to it here. For example:
-Fixes #114

## Changes Made

- [ ] Added new feature
- [ ] Fixed a bug
- [ ] Updated documentation
- [X] Other (please specify)
- Replace NPM install with bun i for setting up test cases

## How to Test

Please describe the steps to test the changes made in this pull request. Include any specific commands or configurations needed.

1. Step 1
2. Step 2
3. Step 3

## Screenshots (if applicable)

If your changes include visual modifications, please add screenshots to help reviewers understand the changes.

## Checklist

- [x] I have read the contributing guidelines
- [x] I have followed the code style guidelines
- [x] My code is tested and works as expected
- [x] I have updated the documentation (if necessary)

## Additional Information

Please provide any additional information that may be helpful for the reviewers.

### CODE REVIEW

File: frontend/lib/store/tests/README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ To run the tests in this project, follow these steps:
 1. **Install Dependencies**  
    Ensure all dependencies are installed. If not, run:
     ```bash
-    npm install
+    bun i
```
Comment (ahaapple) [inline frontend/lib/store/tests/README.md:47]: if install use bun.   the `npm run test`  should also use `bun run test`.  I think we could give both, let the developers choose by themselves

**Comment by Riyan-Mo** [comment]
@ahaapple But using npm to install dependencies gives error. So I guess that's why he suggesting to use bun to install dependencies. This is the error that I encountered.
image

**Comment by ahaapple** [comment]
@Riyan-Mo if so ,we could only use bun. but would better also use bun when run test `bun run test`, which is consistent.

**Comment by Riyan-Mo** [comment]
@ahaapple Yes it would be consistent.

**Comment by Dedsec933** [comment]
@ahaapple i have made the necessary changes

**Review by ahaapple — APPROVED** [review]

**Pull request merged**